src/cryptography/hazmat/backends/openssl/rsa.py

```diff
@@ -106,6 +113,21 @@ def _enc_dec_rsa_pkey_ctx(backend, key, data, padding_enum, padding):
         res = backend._lib.EVP_PKEY_CTX_set_rsa_oaep_md(pkey_ctx, oaep_md)
         backend.openssl_assert(res > 0)
 
+    if (
+        isinstance(padding, OAEP) and
+        padding._label is not None and
+        len(padding._label) > 0
```

Comment:
Why write this condition in a different style than above?

Reply:
Both tests check length now.